### PR TITLE
[OYS-23] Add opening Links in New Window and other attributes to links

### LINKED ISSUE #1688: Opening Links in New WIndow
User Story: As an Open Y content admin, I want the ability to set a link to open in a new window so that I can send users to a new site or PDF file if needed without them losing their place on my Y association site.

Drupal already has a new window function for links available, as evidenced here: 

image

### PR DESCRIPTION
**Pull request by anprok**
Original Issue, this PR is going to fix: https://github.com/ymcatwincities/openy/issues/1688

## Steps for review

- [ ] login as admin
- [ ] visit page /admin/modules
- [ ] make sure that enabled modules Advanced Link, and Link Attributes contrib modules
- [ ] to test link attributes in CKEditor visit /node/add/landing_page
- [ ] add to the content area the Simple content paragraph
- [ ] click on the link button in CKEditor window
- [ ] make sure you see in popup appeared setting to open link in a new window and other attributes

image

- [ ] fill it with any value like http://sandbox.openy.org and enable an open link in new window checkbox
- [ ] add Banner paragraph
- [ ] locate field Link

image

- [ ] fill it with any value like http://sandbox.openy.org and select new window  target in the dropdown
- [ ] fill rest of required fields for paragraph and save the page
- [ ] make sure you see links you added in Simple content and in Banner and they opened in a new window when clicked

### CODE REVIEW

**Comment by podarok** [comment]
This is a feature. @paige-kiecker we shouldn't merge it for the point release, right?

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2075/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2075/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2075/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2075/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2075/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2075/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2075/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2075/

**Review by afi13 — DISMISSED** [review]

**Comment by AndreyMaximov** [comment]
rebuild this please

**Comment by AndreyMaximov** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2078/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2078/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2078/

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_area.yml
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_area.yml:9]: can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_address.yml
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_address.yml:9]: can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_branch_hours.yml
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_branch_hours.yml:8]: can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_coordinates.yml
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_coordinates.yml:8]: can we include https://www.drupal.org/project/datalayer/issues/3102358 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_directions.yml
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_directions.yml:8]: can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_email.yml
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_email.yml:8]: can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_fax.yml
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_fax.yml:8]: can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_phone.yml
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_phone.yml:8]: can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_state.yml
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_state.yml:8]: can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_temp_url.yml
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_temp_url.yml:8]: can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

File: modules/openy_features/openy_node/modules/openy_node_alert/config/install/core.entity_form_display.node.alert.default.yml
Comment (podarok) [inline modules/openy_features/openy_node/modules/openy_node_alert/config/install/core.entity_form_display.node.alert.default.yml]: what happens is more than 10?

Comment (podarok) [reply modules/openy_features/openy_node/modules/openy_node_alert/config/install/core.entity_form_display.node.alert.default.yml]: maybe we should keep it unlimited for not causing issues when 100+ nodes with same title

File: modules/openy_features/openy_prgf/modules/openy_prgf_grid_content/config/install/paragraphs.paragraphs_type.grid_columns.yml
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_grid_content/config/install/paragraphs.paragraphs_type.grid_columns.yml]: can we have a meaningful description for a paragraph?

File: modules/openy_features/openy_prgf/modules/openy_prgf_grid_content/config/install/paragraphs.paragraphs_type.grid_content.yml
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_grid_content/config/install/paragraphs.paragraphs_type.grid_content.yml]: same, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_lto/config/install/paragraphs.paragraphs_type.lto.yml
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_lto/config/install/paragraphs.paragraphs_type.lto.yml]: same, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_promo_card/config/install/paragraphs.paragraphs_type.promo_card.yml
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_promo_card/config/install/paragraphs.paragraphs_type.promo_card.yml]: same, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/core.entity_form_display.paragraph.repeat_schedules.default.yml
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/core.entity_form_display.paragraph.repeat_schedules.default.yml]: do we really need a limit here?

File: modules/openy_features/openy_prgf/modules/openy_prgf_story_card/config/install/paragraphs.paragraphs_type.story_card.yml
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_story_card/config/install/paragraphs.paragraphs_type.story_card.yml]: add a description, please

File: modules/openy_features/openy_prgf/modules/openy_prgf_teaser/config/install/paragraphs.paragraphs_type.teaser.yml
Comment (podarok) [inline modules/openy_features/openy_prgf/modules/openy_prgf_teaser/config/install/paragraphs.paragraphs_type.teaser.yml]: add a description, please

**Review by podarok — CHANGES_REQUESTED** [review]
This is great.
See comments.

Because this is a feature, we should wait until post last point release before the major one for the ability to merge and ship.
Thank you

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_coordinates.yml
Comment (anprok) [reply modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_coordinates.yml:8]: Actually datalayer added to dependencies here because it enabled to expose in third_party_settings below  (expose =1) https://github.com/ymcatwincities/openy/pull/1931/files/17e950a3f51101947824ab8c34eedc767430818a#diff-b3d808e3a07850f153acb9cbc0c49814R12

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_directions.yml
Comment (anprok) [reply modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_directions.yml:8]: Actually datalayer added to dependencies here because it enabled to expose in third_party_settings below  (expose = 1)
https://github.com/ymcatwincities/openy/pull/1931/files/17e950a3f51101947824ab8c34eedc767430818a#diff-fc9e93f1550600cef273b0d5d100a272R12

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_temp_url.yml
Comment (anprok) [reply modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_temp_url.yml:8]: Actually datalayer added to dependencies here because it enabled to expose in third_party_settings below

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_state.yml
Comment (anprok) [reply modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_state.yml:8]: Actually datalayer added to dependencies here because it enabled to expose in third_party_settings below

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_phone.yml
Comment (anprok) [reply modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_phone.yml:8]: Actually datalayer added to dependencies here because it enabled to expose in third_party_settings below

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_fax.yml
Comment (anprok) [reply modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_fax.yml:8]: Actually datalayer added to dependencies here because it enabled to expose in third_party_settings below

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_email.yml
Comment (anprok) [reply modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_email.yml:8]: Actually datalayer added to dependencies here because it enabled to expose in third_party_settings below

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_branch_hours.yml
Comment (anprok) [reply modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_branch_hours.yml:8]: Actually datalayer added to dependencies here because it enabled to expose in third_party_settings below (expose = 1)

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_address.yml
Comment (anprok) [reply modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_address.yml:9]: Actually datalayer added to dependencies here because it enabled to expose in third_party_settings below (expose = 1)

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_area.yml
Comment (anprok) [reply modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_area.yml:9]: Actually datalayer added to dependencies here because it enabled to expose in third_party_settings below. We have expose = 1  here

File: modules/openy_features/openy_prgf/modules/openy_prgf_lto/config/install/paragraphs.paragraphs_type.lto.yml
Comment (anprok) [reply modules/openy_features/openy_prgf/modules/openy_prgf_lto/config/install/paragraphs.paragraphs_type.lto.yml]: Fixed

File: modules/openy_features/openy_prgf/modules/openy_prgf_promo_card/config/install/paragraphs.paragraphs_type.promo_card.yml
Comment (anprok) [reply modules/openy_features/openy_prgf/modules/openy_prgf_promo_card/config/install/paragraphs.paragraphs_type.promo_card.yml]: Fixed

File: modules/openy_features/openy_prgf/modules/openy_prgf_story_card/config/install/paragraphs.paragraphs_type.story_card.yml
Comment (anprok) [reply modules/openy_features/openy_prgf/modules/openy_prgf_story_card/config/install/paragraphs.paragraphs_type.story_card.yml]: Fixed

File: modules/openy_features/openy_prgf/modules/openy_prgf_teaser/config/install/paragraphs.paragraphs_type.teaser.yml
Comment (anprok) [reply modules/openy_features/openy_prgf/modules/openy_prgf_teaser/config/install/paragraphs.paragraphs_type.teaser.yml]: Fixed

File: modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/core.entity_form_display.paragraph.repeat_schedules.default.yml
Comment (anprok) [reply modules/openy_features/openy_prgf/modules/openy_prgf_repeat_schedules/config/install/core.entity_form_display.paragraph.repeat_schedules.default.yml]: Fixed

File: modules/openy_features/openy_node/modules/openy_node_alert/config/install/core.entity_form_display.node.alert.default.yml
Comment (anprok) [reply modules/openy_features/openy_node/modules/openy_node_alert/config/install/core.entity_form_display.node.alert.default.yml]: Fixed

File: modules/openy_features/openy_prgf/modules/openy_prgf_grid_content/config/install/paragraphs.paragraphs_type.grid_content.yml
Comment (anprok) [reply modules/openy_features/openy_prgf/modules/openy_prgf_grid_content/config/install/paragraphs.paragraphs_type.grid_content.yml]: Fixed

File: modules/openy_features/openy_prgf/modules/openy_prgf_grid_content/config/install/paragraphs.paragraphs_type.grid_columns.yml
Comment (anprok) [reply modules/openy_features/openy_prgf/modules/openy_prgf_grid_content/config/install/paragraphs.paragraphs_type.grid_columns.yml]: Fixed

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2098/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2098/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2099/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2099/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2099/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2099/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2099/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2099/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2099/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2098/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2099/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2145/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2145/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2145/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2145/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2145/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2145/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2145/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2145/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2160/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2160/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2160/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2160/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2160/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2160/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2160/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2160/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2171/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2171/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2171/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2171/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2171/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2171/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2171/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2171/

**Review by ddrozdik — APPROVED** [review]

**Comment by ddrozdik** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2191/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2191/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2191/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2191/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2191/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2191/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2191/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2191/

**Comment by ddrozdik** [comment]
@podarok this PR is ready from our side and needs your review. Unfortunately, there is no way to test on build because new builds have no new module.

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2201/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2201/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2201/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2201/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2201/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2201/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2201/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2201/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2208/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2208/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2208/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2208/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2208/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2208/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2208/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2208/

**Pull request merged**
